bigquery-service/src/main/java/com/github/saintukrainian/bigqueryservice/service/LondonCrimeService.java

```diff
@@ -0,0 +1,98 @@
+package com.github.saintukrainian.bigqueryservice.service;
+
+import com.github.saintukrainian.bigqueryservice.entities.LondonCrime;
+import com.google.cloud.bigquery.*;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Slf4j
+@Service
+public class LondonCrimeService extends BigQueryService {
+
+  @Value("${sql.london-crimes.by.year.limit-10}")
+  private String LONDON_CRIMES_MAX_10;
+
+  @Value("${sql.london-crimes.by.popularity.year.limit-1}")
+  private String LONDON_CRIME_MOST_POPULAR;
+
+  public List<LondonCrime> getLondonCrimesByYear(int year) throws InterruptedException {
+    List<LondonCrime> crimesByYear = new ArrayList<>();
+    QueryJobConfiguration queryConfig =
+        QueryJobConfiguration.newBuilder(LONDON_CRIMES_MAX_10)
+            .addNamedParameter("year", QueryParameterValue.int64(year))
+            .setUseLegacySql(false)
```

Comment:
As far as I know legacy sql is false by default, so probably it is redundant to specify it here, but please double check this.